components/articles/Article.js

```diff
@@ -19,6 +19,8 @@ import { ROUTES_NAMES } from 'routes';
 
 const COVER_SIZES = [1200, 1000, 770, 640, 360];
 
+const AUTHOR_BRAND_THUMBNAIL_WIDTH = 100;
```

Comment:
but shown as `48px`

Reply:
48px is for square images. For some wide logos (like Art Museum) width is actually more than 48px. The height is 48px though